go.mod

```
@@ -1,4 +1,4 @@
-module github.com/aws/karpenter-core
+module github.com/sigs.k8s.io/karpenter
```

```suggestion
module sigs.k8s.io/karpenter
```

There's no Github prefix to the module name